pkg/gitreceive/storage/slug_builder_info.go

```diff
@@ -18,7 +18,7 @@ type SlugBuilderInfo struct {
 func NewSlugBuilderInfo(s3Endpoint, appName, slugName string, gitSha *git.SHA) *SlugBuilderInfo {
 	tarKey := fmt.Sprintf("home/%s/tar", slugName)
 	// this is where workflow tells slugrunner to download the slug from, so we have to tell slugbuilder to upload it to here
-	pushKey := fmt.Sprintf("home/%s:git-%s/push", appName, gitSha.Full())
+	pushKey := fmt.Sprintf("home/%s:git-%s/push", appName, gitSha.Short())
```

Comment:
Does this short sha change correspond with the short sha change in the buildhook [above](https://github.com/deis/builder/pull/129/files#diff-91e10a0836e513c85626cee1d2899e5dR212)?